docs-src/guide-rpc-server-api.scrbl

```diff
@@ -0,0 +1,333 @@
+@section[#:tag-prefix "non-stdlib" #:tag "routes"]{Non-stdlib routes}
+
+As mentioned before, there are other parent RPC routes than just
+@urlin{/stdlib}. These route namespaces have special significance as described
+below.
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "acc-deploy"]{/acc/deploy}
+
+@margin-note{
+Q: What happened to the @jsin{backend} argument when calling @tech{account}
+methods via RPC?
+
+A: The server handles this so clients don't have to!
+}
+
+The @urlin{/acc/deploy} route corresponds to a @jsin{deploy} operation on
+@tech{accounts} when using the standard library directly.
+
+@(define link-deploy-stdlib @reachexlink[
+  #:ref "27fb2a9"
+  #:loc (cons "11" "12")
+  "maybe-send/index.mjs"
+  "this code from the \"maybe-send\""])
+
+@(define link-deploy-rpc @reachexlink[
+  #:ref "1474d32"
+  #:loc "97"
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+Contrast @link-deploy-stdlib example:
+@js{
+  console.log(`Alice will deploy the contract.`);
+  const ctcAlice = alice.deploy(backend);
+}
+
+...with the equivalent @link-deploy-rpc Python example:
+@py{
+  ctc_alice = rpc('/acc/deploy', acc_alice)
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "acc-attach"]{/acc/attach}
+
+The @urlin{/acc/attach} route corresponds to an @jsin{attach} operation on
+@tech{accounts} when using the standard library directly.
+
+@(define link-attach-stdlib @reachexlink[
+  #:ref "27fb2a9"
+  #:loc (cons "14" "15")
+  "maybe-send/index.mjs"
+  "this code from the \"maybe-send\""])
+
+@(define link-attach-rpc @reachexlink[
+  #:ref "1474d32"
+  #:loc "98"
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+Contrast @link-attach-stdlib example:
+@js{
+  console.log(`Bob will attach to the contract.`);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+}
+
+...with the equivalent @link-attach-rpc Python example:
+@py{
+  ctc_bob = rpc('/acc/attach', acc_bob, rpc('/ctc/getInfo', ctc_alice))
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "ctc-getinfo"]{/ctc/getInfo}
+
+The @urlin{/ctc/getInfo} route corresponds to a @jsin{getInfo} operation on
+@tech{contracts} when using the standard library directly.
+
+Contrast @link-attach-stdlib example:
+@js{
+  console.log(`Bob will attach to the contract.`);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+                                     ------------------
+}
+
+...with the equivalent @link-attach-rpc Python example:
+@py{
+  ctc_bob = rpc('/acc/attach', acc_bob, rpc('/ctc/getInfo', ctc_alice))
+                                        ------------------------------
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "forget-acc"]{/forget/acc}
+See: @secref{id-clean-up}.
```

Comment:
You shouldn't have sections like this... this should probably be the id-clean-up section or the routes should be listed there